tests/baselines/reference/aliasOnMergedModuleInterface.errors.txt

```diff
@@ -8,7 +8,7 @@ tests/cases/compiler/aliasOnMergedModuleInterface_1.ts(5,16): error TS2304: Cann
     z.bar("hello"); // This should be ok
     var x: foo.A = foo.bar("hello"); // foo.A should be ok but foo.bar should be error
                    ~~~
-!!! error TS2304: Cannot find name 'foo'.
+!!! error TS2691: Cannot find name 'foo'. A type exists with this name, but no value.

```

Comment:
But this current example should be reverted. You might need to "chase down" the alias (i.e. keep resolving things the alias refers to until you get to a non-alias entity.